VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -295,8 +295,13 @@ def try_parse_context(self, seqNo):
             self.logging_file=self._context._log_file
             self._context._shell_log_file = os.path.join(handler_env['handlerEnvironment']['logFolder'],'shell.log')
             self._change_log_file()
-            self._context._event_dir = handler_env['handlerEnvironment']['eventsFolder']
-            self.event_dir = self._context._event_dir
+            try:
+                if(self.get_intvalue_from_configfile("disable_logging", 0) == 0):
```

Comment:
nit: all other places seem to use ' instead of " for describing literal strings. Can we keep it consistent?